src/main/java/frc/robot/Constants.java

```diff
@@ -24,9 +26,10 @@
     public static final int REVERSE_INTAKE = 3;
     public static final int FEED_CONVEYOR = 1;
     public static final int REVERSE_CONVEYOR = 3;
-    public static final int CHARGE_SHOOTER = 2;
+    public static final int CHARGE_SHOOTER = 5;
 
-    public static final int TOGGLE_RINGLIGHT = 5;
+    public static final int TOGGLE_VISION = 5;
+    public static final int RUN_AIMBOT = 2;
```

Comment:
It makes more sense to put these in the `Driver` class.  Since it will get confusing when there is an operator class with the same buttons.  Also it is not immediately apparent that these are button numbers but moving it to `Driver` will eliminate this confusion.